fluentd/configs.d/filter-pre-mux-client.conf

```diff
@@ -1,7 +1,7 @@
 <match **>
   @type secure_forward
   log_level "#{ENV['FORWARD_INPUT_LOG_LEVEL'] || ENV['LOG_LEVEL'] || 'warn'}"
-  self_hostname ${HOSTNAME}
+  self_hostname ${hostname}
```

Comment:
shouldn't this remain `HOSTNAME` so that we can get the name of the pod that's injected into our ENV by k8s? or does `hostname` also resolve to that on the pod?

Reply:
The lowercase ${hostname} is a placeholder provided by fluentd.  Please see this bug.  Thanks!
https://bugzilla.redhat.com/show_bug.cgi?id=1548720

Reply:
ack. that's strange though, when did that stop working? When we originally introduced secure forward `HOSTNAME` resolved correctly...

Reply:
Not really sure what `self_hostname` is used for, access control maybe? but we aren't using that

Reply:
It looks it's being used at least for checking the forwarder vs the forwardee?  (Please note that @anpingli added just the line 176 to reveal the bug.)
https://bugzilla.redhat.com/show_bug.cgi?id=1548720#c0